app/src/features/refusjon-omsorgspenger-arbeidsgiver/Steg2AnsattOgArbeidsgiver.tsx

```diff
@@ -6,16 +6,34 @@ import {
   Button,
   Heading,
   Label,
+  Select,
+  Skeleton,
   TextField,
 } from "@navikt/ds-react";
+import { useQuery } from "@tanstack/react-query";
 import { Link } from "@tanstack/react-router";
+import { useState } from "react";
 
 import { Informasjonsseksjon } from "~/features/Informasjonsseksjon";
 import { RotLayout } from "~/features/rot-layout/RotLayout";
+import { navnMedStorBokstav } from "~/utils";
 
+import { slåOppPersondataOptions } from "./api/queries";
 import { Fremgangsindikator } from "./Fremgangsindikator";
 
 export const RefusjonOmsorgspengerArbeidsgiverSteg2 = () => {
+  const [fødselsnummer, setFødselsnummer] = useState("");
+  const { data, error, isLoading } = useQuery(
+    slåOppPersondataOptions(fødselsnummer),
+  );
+
+  const fantIngenPersoner =
```

Comment:
Kunne denne bare vært `error?.feilkode === "fant ingen personer";` ?

Reply:
Nope, for error kan også være en Error type, og den typen har ikke "feilkode" feltet, så da må jeg sjekke eksplisitt.